src/components/Grid/Grid.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { Cell } from "components/Cell";
+import './Grid.css'
+import { Provider } from "react-redux";
+import configureStore from "configureStore";
+
+
+const Grid = (props) => {
+
+  return (
+    <Provider store={configureStore()}>
+      <div className='grid'>
+        {(() => {
+          const grid = [];
+          for (let row = 0; row < 50; row++) {
+            grid.push([])
+            for (let col = 0; col < 50; col++) {
+              grid[row].push(<Cell />)
+            }
+          }
+          return grid;
+        })()}
+      </div>
```

Comment:
Ici tu construits ta grille dans ton composant et tu utilises des boucles for.

Nous avons par convention de limiter/prohiber la déclaration de variable avec `let` en  javascript et de toujours utiliser des contants (que l'on déclare avec `const`.

L'idée va donc être:
Dans un premier temps de générer un tableau à double entrées contenant des booléean dans l'initiale state de ton reducer game (objet game de ton store redux)
Puis de récupérer ce tableau dans ton composant avec un useSelector
Et enfin d'utiliser la méthode `Array.map` pour transformer chaque booléen en un appel de ton composant `Cell`






 


et d'utiliser un .map sur ton tableau pour afficher une cellule poour chacune de ces valeurs.